src/contexts/UserContext.js

```diff
@@ -51,15 +56,19 @@ class UserProvider extends React.Component {
       .then((res) => {
         console.log(res);
         const userReturn = res.data.user.username;
```

Comment:
If there is an error response, is the promise rejected? If not the status code should be checked before accessing response values and writing them to storage.

Reply:
I have a catch set up after, which admittedly I'm not doing anything with the error other than logging it, but should I do something before the res?

Reply:
It's fine for now, but eventually we should come up with an error handling approach that allows the app to respond to errors in addition to reporting them to the console.